components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -39,6 +39,7 @@
 #include "brave/common/pref_names.h"
 #include "brave/components/brave_ads/browser/ads_service.h"
 #include "brave/components/brave_ads/browser/ads_service_factory.h"
+#include "brave/components/brave_ads/browser/buildflags/buildflags.h"
```

Comment:
is there already a dep for this? If not please add one

Reply:
Yes it is.

Reply:
I don't see one in `//brave/components/brave_rewards/browser`

Reply:
yeah, my bad, missed it.